publicsuffixlist.js

```diff
@@ -489,9 +489,9 @@ const suffixInPSL = function(hostname) {
 
 /******************************************************************************/
 
-const toSelfie = function(encoder) {
+const toSelfie = function(encoder = null) {
     if ( pslBuffer8 === undefined ) { return ''; }
-    if ( encoder instanceof Object ) {
```

Comment:
This was problematic anyway. First of all `Object` is a global. Another script in the same document could redefine it. Secondly `[] instanceof Object === true`.